samples/powerplay/src/main/cpp/PowerPlayMultiPlayer.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <android/log.h>
+#include "PowerPlayMultiPlayer.h"
+
+static const char *TAG = "PowerPlayMultiPlayer";
+
+using namespace oboe;
+using namespace parselib;
+
+PowerPlayMultiPlayer::PowerPlayMultiPlayer() = default;
+
+PowerPlayMultiPlayer::~PowerPlayMultiPlayer() = default;
+
+void PowerPlayMultiPlayer::MyPresentationCallback::onPresentationEnded(oboe::AudioStream *oboeStream) {
+    __android_log_print(ANDROID_LOG_INFO, TAG,
+                        "==== MyPresentationCallback() called with gain %f",
+                        mParent->getGain(0)
+    );
+}
+
+void PowerPlayMultiPlayer::setupAudioStream(int32_t channelCount, oboe::PerformanceMode performanceMode) {
+    __android_log_print(ANDROID_LOG_INFO, TAG, "setupAudioStream()");
+    mChannelCount = channelCount;
```

Comment:
Shouldn't the channel count be read from the file header?

Reply:
Ideally. For now though, let's just assume stereo until we implement spacial audio for  the sake of simplcity and this PR is getting to large.